### PR TITLE
remove unneeded code

### PR DESCRIPTION
**Pull request by piotr-s-brainhub**
- [ ] `package.json:version` has been updated with `npm version patch` (or `major/minor` as appropriate)
- [ ] `@brainhubeu/react-carousel` version has been updated in `docs-www/package.json` to the same which is in `package.json:version`

### CODE REVIEW

File: .nvmrc [hunk 1]
```diff
@@ -1 +0,0 @@
-9
```
Comment (RobertHebel) [inline .nvmrc:1]: I wouldn't say its unneeded, rather it's outdated. According to nvm [docs](https://github.com/nvm-sh/nvm#nvmrc)

>You can create a .nvmrc file containing a node version number (or any other string that nvm understands; see nvm --help for details) in the project root directory (or any parent directory). Afterwards, nvm use, nvm install, nvm exec, nvm run, and nvm which will use the version specified in the .nvmrc file if no version is supplied on the command line.

Comment (piotr-s-brainhub) [reply .nvmrc:1]: @RobertHebel 

so maybe we could add `"node"` [here](https://github.com/brainhubeu/react-carousel/blob/master/package.json#L6), so we could use locally any Node.js version higher or equal than the minimum version?

Comment (RobertHebel) [reply .nvmrc:1]: 👍

Comment (piotr-s-brainhub) [reply .nvmrc:1]: @RobertHebel 

done in 6df7fd00f91e1f11aa4589110ee2e4404b4ebe3c

**Pull request merged**